### DISCUSSION TITLE
Can MiniScaffold rely on FAKE?

### DISCUSSION THREAD

**Discussion by halcwb** [TheAngryByrd/MiniScaffold#240] [General]
**Is your feature request related to a problem? Please describe.**
MiniScaffold heavily relies on FAKE for the "Make" scripting. However, FAKE seems to be a bit in trouble because of difficulty to maintain the project. See my recent issue regarding a problem with FAKE using Paket for new net5 windows projects: https://github.com/fsharp/FAKE/issues/2564.

**Describe the solution you'd like**
Of course this has serious consequences for MiniScaffold as well. Either the maintenance problem of FAKE get's solved, or MiniScaffold should look at a different FAKE independent scaffolding setup.

**Describe alternatives you've considered**
I have tried to pinpoint the problem I had as precisely as possible, however, I got no response from the FAKE maintainers yet.

@TheAngryByrd Your thoughts?

**Comment by TheAngryByrd** [comment]
I think this bigger question than MiniScaffold.  Can F# ecosystem live without FAKE?  I know many project use it so this would be a big disruption to much of the ecosystem as a whole.

As far as MinIScaffold goes... yes it would affect this project but I don't know what the alternative would be without lots of major rework.

**Comment by halcwb** [comment]
That was my thought as well, can F# afford losing FAKE? I am considering a post about this issue in the F# forum.

**Comment by TheAngryByrd** [comment]
Yeah I think that would be a good idea. I've considered bringing it up on the F# Foundation Slack as well.

**Comment by halcwb** [comment]
I put my thoughts on the [forum](https://forums.fsharp.org/t/can-f-afford-losing-fake-and-other-libraries/1642).

**Comment by TheAngryByrd** [comment]
Seems like FAKE has gotten a lot of love and just released 6.0.
